docs/griptape-cloud/structures/create-structure.md

```diff
@@ -0,0 +1,16 @@
+# Structures
+
+Use [Structures](https://cloud.griptape.ai/structures) to host any python code in the cloud.
```

Comment:
When I read the docs preview, I feel like I want to click on this link to jump to the definition of what a structure is, rather than jump to the structures page in the ui. I could see use cases for both. Could we have something that calls out the two things individually?

Something that conveys:
1. What is a structure? (provide a definition or link to glossary of common cloud concepts)
2. Where are my structures? (provide a link to "my structures" or something like that)

Reply:
Added a sentence on what each of these things are. We can iterate on those descriptions as we see fit.